doc/whats_new/v0.24.rst

```diff
@@ -827,3 +830,57 @@ Code and Documentation Contributors
 
 Thanks to everyone who has contributed to the maintenance and improvement of
 the project since version 0.23, including:
+
+Abo7atm, Adam Spannbauer, Adrin Jalali, adrinjalali, Agamemnon Krasoulis,
```

Comment:
This change shouldn't be in the same PR as changing the release data, to be able to easily pull it into the release branch once merged.

Reply:
This was already merged in 0.24.X. We can always update if necessary when 0.24.0 final is out.

Reply:
I see, I guess I have taken them in one PR just for the consistency of the record. Good then :)